exercises/concept/plane-tickets/.meta/exemplar.py

```diff
@@ -0,0 +1,44 @@
+""" Plane Tickets Exercise """
+
+def generate_seats(amount):
+    
+    """ Generate a series of seat numbers for airline boarding.
+    
+    :param amount: Amount of seats to be generated. (int)
+    :return: Iterable that generates seat numbers. (generator)
+    
+    There should be no row 13
+    
+    Seat numbers are generated with each row having 4 seats.
+    These should be sorted from low to high.
+    
+    Example: 3C, 3D, 4A, 4B
+    
+    """
+    
+    SEATS_IN_ROW = ["A", "B", "C", "D"]
+    
+    amount = amount+4 if amount >= 13 else amount
+    
+    for seat in range(amount):
+        row_number = -(-(seat+1) // 4)          # ? Ceiling division; might be too advanced for students?
+        if row_number != 13:
+            seat_letter = SEATS_IN_ROW[seat % 4]
+            yield str(row_number)+str(seat_letter)
```

Comment:
Extraneous `str()` call performed on a string? f-strings?
Would it be cleaner to just use a nested loop?

Reply:
```suggestion
            yield str(row_number) + seat_letter
```
I mean yes, we could use f-strings, but that would IMO not look that much cleaner. I love f-strings though.

With nested loops, do you mean comprehensions?

Reply:
```
num = 1
count = 0
while True:
    for letter in "ABCD":
        yield f"{num}{letter}"
        count += 1
        if count > amount:
            return
    count += 1
    if count == 13:
        count += 1
````

Reply:
I mean it is an exemplar for mentors, and I mean it is not something they haven't necessarily learned?